server/middleware/SchemaValidator.js

```diff
@@ -22,9 +22,34 @@ const validation = {
         status: 400,
         error: result.error.details[0].message
       });
-    } else {
+    } 
       next();
-    }
   }
-};
-export default validation;
+  pathVaridete(req, res, next){
+    const schema = {
+      id: Joi.number().required()
+    };
+    const result = Joi.validate(req.params, schema);
+    if (result.error) {
+     return res.status(400).send({
+        status: 400,
+        error: result.error.details[0].message
+      });
+    } 
+    next();
+  }
+  pathValidate(req, res, next){
```

Comment:
What's the difference between these 2 methods (athVaridete and pathValidate), I believe we can just have 1 method as opposed to having 2 since they also serve the same purpose.